rest/rest-cli/src/test/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/SubmitJobCommandTest.java

```diff
@@ -105,12 +105,12 @@ public void init() {
         Mockito.when(currentContextMock.resultStack()).thenReturn(stack);
     }
 
-    @Test(expected = CLIException.class)
-    public void testNoFilePathProvided() throws Exception {
+    @Test(expected = NullPointerException.class)
+    public void testNoFilePathProvided() throws Exception, NullPointerException {
 
         params = new String[1];
         params[0] = new String();
-        params[0] = "";
+        params[0] = null;
 
```

Comment:
Not sure I understand here, do you mean params[1] = null;?
Then one question to think about for params[0] = new String(), do you want a variable or a constant here?